### PR TITLE
Add sample to update a cloud monitoring uptime check.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**

### CODE REVIEW

File: monitoring/api/v3/uptime-check-client/snippets.py [hunk 1]
```diff
@@ -42,6 +42,22 @@ def create_uptime_check_config(project_name, host_name=None,
...
+    client = monitoring_v3.UptimeCheckServiceClient()
+    config = client.get_uptime_check_config(config_name)
+    field_mask = monitoring_v3.types.FieldMask()
+    if new_display_name:
+        field_mask.paths.append('display_name')
+        config.display_name = new_display_name
+    if new_http_check_path:
+        field_mask.paths.append('http_check.path')
+        config.http_check.path = new_http_check_path
+    client.update_uptime_check_config(config, field_mask)
```
Comment (tswast) [inline monitoring/api/v3/uptime-check-client/snippets.py]: Aside: I find it slightly annoying that the autogen client doesn't take a list of strings for `field_mask`.

File: monitoring/api/v3/uptime-check-client/snippets.py [hunk 2]
```diff
@@ -170,3 +203,11 @@ def project_name():
 
     elif args.command == 'delete-uptime-check-config':
         delete_uptime_check_config(args.name)
+
+    elif args.command == 'update-uptime-check-config':
+        if not args.display_name and not args.uptime_check_path:
+            print("Nothing to update.  Pass --display_name or "
```
Comment (tswast) [inline monitoring/api/v3/uptime-check-client/snippets.py]: Nit: single-quotes for strings.

File: monitoring/api/v3/uptime-check-client/snippets.py (hunk 1 above)
Comment (SurferJeffAtGoogle) [reply monitoring/api/v3/uptime-check-client/snippets.py]: Agreed.

File: monitoring/api/v3/uptime-check-client/snippets.py (hunk 2 above)
Comment (SurferJeffAtGoogle) [reply monitoring/api/v3/uptime-check-client/snippets.py]: Fixed.

**Pull request merged**